flood_api/routers/flood.py

```diff
@@ -22,7 +22,11 @@
 
 
 @router.get(
-    "/summary", response_model=SummaryResponseModel, response_class=JSONResponse
+    "/summary",
+    summary="Get summary forecast",
+    description="Returns a summary forecast of the next 30 days for the given location",
+    response_model=SummaryResponseModel,
+    response_class=JSONResponse,
```

Comment:
What does specifying `response_class=JSONResponse` do?

Reply:
Nothing since `JSONResponse` is the default

Reply:
In that case I think it's better without